src/alire/alire-publish.adb

```diff
@@ -255,12 +256,28 @@ package body Alire.Publish is
          Log_Success
            ("Your index manifest file has been generated at "
             & TTY.URL (Index_Manifest));
-         Log_Success
-           ("Please upload this file to "
-            & TTY.URL
-              (Index.Community_Upload_URL
-               & "/" & TOML_Index.Manifest_Path (Name))
-            & " to create a pull request against the community index.");
+
+         --  Show the upload URL in normal circumstances, or a more generic
+         --  message otherwise (when lacking a github login).
+
+         if Config.Defined (Config.Keys.User_Github_Login) then
```

Comment:
Why is there an alternative here? The user need a GitHub account to contribute to the index anyway.

Reply:
This one is because when `--force` is in effect we skip the checks about github, so it's possible to reach this stage without a configured github login, if you never run `alr init` before.

I did it like this to allow our tests to succeed locally, and to allow people that wants to publish to other indexes besides the community one, to be able to get the manifest without requiring a real github account. But then it's true the message for that case might not be appropriate.